andela/ah-bird-box-frontend
# #163388206 Update TravisCI to run tests

**What does this PR do?**
Integrates Travis for continuous testing and integration. Adds a Travis readme badge to the repo.

**Description of Task to be completed?**
1. Add .travis.yml file
2. Include test commands
3. Add the repo to travis-ci.com

**How should this be manually tested?**
- Make a commit to the repo
- check the repo building with tests at https://travis-ci.com/

What are the relevant pivotal tracker stories?
[#163388206](https://www.pivotaltracker.com/story/show/163388206)
